packages/frontend/web-ui/src/auth/pages/Login.spec.tsx

```diff
@@ -131,34 +132,33 @@ describe(Login.name, () => {
         setFormField: setFormFieldMock,
       });
 
-      useShowPassword.mockReturnValue({
+      (useShowPassword as jest.Mock<typeof useShowPassword>).mockReturnValue({
         showPassword: false,
         handleClickShowPassword: handleClickShowPasswordMock,
         handleMouseDownPassword: handleMouseDownPasswordMock,
       });
 
-      useSelector.mockImplementation(() => ({
-        token: tokenFixture,
-        errorMessage: errorMessageFixture,
-      }));
+      (
+        useAppSelector as unknown as jest.Mock<typeof useAppSelector>
+      ).mockReturnValueOnce(tokenFixture);
 
-      render(
+      const renderResult: RenderResult = render(
         <MemoryRouter>
           <Login />
         </MemoryRouter>,
       );
 
-      const formEmailTextField = screen.getByLabelText('form-login-email');
-      const formEmailTextFieldP = formEmailTextField.querySelector('p');
+      const formEmailTextFieldParagraph: Text | null =
+        (renderResult.container.querySelector('.form-login-email')
+          ?.childNodes[2]?.firstChild as Text) ?? null;
 
-      pErrorEmail = formEmailTextFieldP.innerHTML;
+      pErrorEmail = formEmailTextFieldParagraph?.textContent;
 
-      const formPasswordTextField = screen.getByLabelText(
-        'form-login-password',
-      );
-      const formPasswordTextFieldP = formPasswordTextField.querySelector('p');
+      const formPasswordTextFieldParagraph: Text | null =
+        (renderResult.container.querySelector('.form-login-password')
+          ?.childNodes[2]?.firstChild as Text) ?? null;
 
-      pErrorPassword = formPasswordTextFieldP.innerHTML;
+      pErrorPassword = formPasswordTextFieldParagraph?.textContent;
     });
 
     afterAll(() => {
```

Comment:
`useLoginForm`, `useShowPassword` must be reset here if we use `mockReturnValue`.

Reply:
Done!